torchx/specs/named_resources_aws.py

```diff
@@ -348,6 +348,46 @@ def aws_trn1_32xlarge() -> Resource:
     )
 
 
+def aws_inf2_xlarge() -> Resource:
+    return Resource(
+        cpu=4,
+        gpu=0,
+        memMB=32 * GiB,
+        capabilities={K8S_ITYPE: "inf2.xlarge"},
+        devices={NEURON_DEVICE: 1},
+    )
+
+
+def aws_inf2_8xlarge() -> Resource:
+    return Resource(
+        cpu=32,
+        gpu=0,
+        memMB=32 * GiB,
+        capabilities={K8S_ITYPE: "inf2.8xlarge"},
+        devices={NEURON_DEVICE: 1},
+    )
+
+
+def aws_inf2_24xlarge() -> Resource:
+    return Resource(
+        cpu=96,
+        gpu=0,
+        memMB=192 * GiB,
```

Comment:
384 GB of RAM

Reply:
Even though it's 384 according to the spec: https://aws.amazon.com/ec2/instance-types/inf2/
In reality we have to use smaller number or we will get this from Batch:
```
Invalid memory value. It should be within the range [1, 376832]
```
With the current MEM_TAX=0.96:
`376832/(384*1024*0.96) = 0.99826388888` => thus it has to be at most 383 with the current mem tax

Reply:
@clumsy  This is good insights - I gave it a bit more thoughts, I prefer to have this config to stick to actual values of the instances, and have this mem_tax logic somewhere else, so that it can apply to other instances as well. 

Also, probably it depends on your setup, we did observe similar issues on inf2.48xlarge batch queue tho.

Reply:
I'm OK with any solution that works. This right now I'm not sure works...